### DISCUSSION TITLE
Math Expressions Support for GitHub Enterprise Server

### DISCUSSION THREAD

**Discussion by ptasev** [community/community#17969] [Enterprise] [answered]
GitHub recently added math expression render support in markdown. This request is to get the same feature for GitHub Enterprise Server.

https://github.blog/changelog/2022-05-19-render-mathematical-expressions-in-markdown/

**Comment by davidstaheli** [comment, marked as answer]
Hi, @ptasev! We plan to release mathematical expression support (as well as Mermaid diagram support) in GHES 3.7 in approximately November 2022. Thanks for your patience!
